Sources_v3/Query/Pagination.swift

```diff
@@ -4,67 +4,60 @@
 
 import Foundation
 
-public typealias Pagination = Set<PaginationOption>
+public extension Int {
```

Comment:
What should we do with this? 🙂

Reply:
I think we should think about the value if they make sense but, in general, I'd keep the constants around

Reply:
Ok, I removed `nextPage***` constants cause we haven't even used them and kept others.